Automattic/newspack-listings
# fix(self-serve): patterns class name after refactor

### All Submissions:






### Changes proposed in this Pull Request:



Fixes a fatal error due to a misnamed class. #233 refactored class names and namespaces throughout the plugin, but there were a few lingering instances of the old class names for the `\Newspack_Listings\Block_Patterns` class in the self-serve purchase flow.

### How to test the changes in this Pull Request:

1. On `master`, set up self-serve listings and attempt to complete a test purchase. Observe that you can't due to a 500 error:

```
Uncaught Error: Class 'Newspack_Listings\Patterns' not found in /wp-content/plugins/newspack-listings/includes/products/class-products-purchase.php:392
```

2. Check out this branch and repeat. (You'll need to start over, from the purchase form again.) Confirm that the purchase now completes processing successfully without error, and that you can edit, preview, and manage your listing via the My Account dashboard as expected.

### Other information:
